src/accordion/Accordion.test.js

```diff
@@ -0,0 +1,46 @@
+import test from 'ava';
+import jsdom from 'jsdom';
+
+import { Accordion } from './Accordion';
+
+global.document = jsdom.jsdom(`
+  <div id="accordion">
+    <ul class="accordion-list">
+      <li class="accordion-list-item">
+        <div class="accordion-section js-accordion-section">
+          <div class="accordion-section-toggle js-accordion-section-toggle">
+            <h4 class="accordion-section-toggle-content">Header</h4>
+          </div>
+          <ul class="accordion-section-list js-accordion-section-content">
+            <li class="accordion-section-list-item">
+              Content
+            </li>
+          </ul>
+        </div>
+      </li>
+      <li class="accordion-list-item">
+        <div class="accordion-section js-accordion-section">
+          <div class="accordion-section-toggle js-accordion-section-toggle">
+            <h4 class="accordion-section-toggle-content">Header</h4>
+          </div>
+          <ul class="accordion-section-list js-accordion-section-content">
+            <li class="accordion-section-list-item">
+              Content
+            </li>
+          </ul>
+        </div>
+      </li>
+    </ul>
+  </div>`);
```

Comment:
@stewartyu Currently the precompile task that creates js templates happens in nightshade. One approach, which seems the cleanest,  is to add nunjucks to nightshade-core and a template file to each pattern. Then in tests we just call `nunjucks.render` and pass in the template.